smk/gene_annotation.smk

```diff
@@ -361,29 +360,24 @@ rule gene_annot_dbcan:
         config["minto_dir"]+"/envs/gene_annotation.yml"
     shell:
         """
-        if [[ ! -f {params.dbcan_db}/fam-substrate-mapping.tsv ]]; then
-          echo "Error: dbCAN database needs to be re-built, run dependencies.smk" > {log};
-          exit 1;
-        fi
-
         time (
-            run_dbcan {input.faa} protein --db_dir {params.dbcan_db} --dia_cpu {threads} --out_pre dbcan_ --out_dir out
-            echo -e "#Database downloaded\t$(conda list | sed -E 's|[[:space:]]+| |g' | cut -d' ' -f 1-2 | grep -P dbcan)\t$(stat -c '%y' {params.dbcan_db}/fam-substrate-mapping.tsv | cut -d' ' -f 1)" > dbcan_processed.txt
+            run_dbcan {input.faa} protein --db_dir $(dirname {input.dbcan_db}) --dia_cpu {threads} --out_pre dbcan_ --out_dir out
+            echo -e "#Database downloaded\\t$(conda list | sed -E 's|[[:space:]]+| |g' | cut -d' ' -f 1-2 | grep -P dbcan)\\t$(stat -c '%y' {input.dbcan_db} | cut -d' ' -f 1)" > dbcan_processed.txt
             {script_dir}/process_dbcan_overview.pl out/dbcan_overview.txt >> dbcan_processed.txt
             rsync -a dbcan_processed.txt {output}
         ) >& {log}
         """
 
 rule gene_annot_eggnog:
     input:
-        faa=rules.rename_prokka_sequences.output.faa
+        faa=rules.rename_prokka_sequences.output.faa,
+        eggnog_db="{minto_dir}/data/eggnog_data/data/eggnog.db"
```

Comment:
Same here: `minto_dir` is not a wildcard in output.

Reply:
Fixed these